1/contrib/jenkins/run-jenkins

```diff
@@ -6,24 +6,17 @@
 #
 # It also sets the admin password to ${JENKINS_PASSWORD}.
 #
+source /usr/local/bin/jenkins-common.sh
 
-# Generate passwd file based on current uid
-function generate_passwd_file() {
-  export USER_ID=$1
-  export GROUP_ID=$2
-  envsubst < /opt/openshift/passwd.template > /opt/openshift/passwd
-  export LD_PRELOAD=libnss_wrapper.so
-  export NSS_WRAPPER_PASSWD=/opt/openshift/passwd
-  export NSS_WRAPPER_GROUP=/etc/group
-}
+for name in $(get_is_names); do
```

Comment:
meaning both "i can run this image outside openshift but have it connect to an openshift cluster" and "i can run this image outside openshift with no intent to use it with openshift"